tests/test_flow.py

```diff
@@ -23,6 +23,7 @@
     ParquetTarget, QueryByKey, \
     TSDBTarget, Extend, SendToHttp, HttpRequest, NoSqlTarget, NoopDriver, Driver, Recover, V3ioDriver, ParquetSource
 from storey.flow import _ConcurrentJobExecution, Context, ReifyMetadata, Rename
+from integration.integration_test_utils import append_return
```

Comment:
I'm pretty sure there's no need to change this file.

Reply:
fixed